Apps/AdminWebClient/src/ClientApp/src/views/Communication.vue

```diff
@@ -86,15 +97,79 @@ export default class CommunicationView extends Vue {
+        },
+        {
+            text: "Effective On",
+            value: "effectiveOn",
+            width: "20%"
+        },
+        {
+            text: "Expires On",
+            value: "expiresOn",
+            width: "20%"
+        },
+        {
+            text: "Text",
+            value: "text",
+            width: "40%"
+        }
+    ];
+
+    private bannerList: Communication[] = [];
+
   private communicationService!: ICommunicationService;
 
   mounted() {
     this.communicationService = container.get(
       SERVICE_IDENTIFIER.CommunicationService
     );
-    this.clearForm();
+      this.clearForm();
+      this.loadBannerList();
   }
 
+    private loadBannerList() {
+
+        // Dummy data for testing banner list
+        //  This will need to be disabled when we hook this to the real service via vue service.
+        this.bannerList = [{ id: 'b1001', subject: 'banner 1', text: 'B1 - Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum s', effectiveDateTime: new Date('2020-08-05'), expiryDateTime: new Date('2020-08-07') },
+            { id: 'b1002', subject: 'banner 2', text: 'B2 - Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum s', effectiveDateTime: new Date('2020-06-30'), expiryDateTime: new Date('2020-07-01') },
+            { id: 'b1003', subject: 'banner 3', text: 'B3 - Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum s', effectiveDateTime: new Date('2020-05-03'), expiryDateTime: new Date('2020-05-28') },
+            { id: 'b1004', subject: 'banner 4', text: 'B4 - Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum s', effectiveDateTime: new Date('2020-07-30'), expiryDateTime: new Date('2020-08-01') },
+            { id: 'b1005', subject: 'banner 5', text: 'B5 - Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum s', effectiveDateTime: new Date('2020-06-23'), expiryDateTime: new Date('2020-06-25') },
+
+            { id: 'b10012', subject: 'banner 1', text: 'B1 - Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum s', effectiveDateTime: new Date('2020-06-18'), expiryDateTime: new Date('2020-07-20') },
+            { id: 'b10052', subject: 'banner 52', text: 'B5 - Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum s', effectiveDateTime: new Date('2020-06-15'), expiryDateTime: new Date('2020-06-17') }
+        ];
+
+        //Todo for 8213: Enable the following when 8213 update vue service is complete.
+        //this.communicationService
+        //    .getBannerList()
+        //    .then(banners => {
+        //        this.bannerList = [];
+        //        this.bannerList.push(...banners); 
```

Comment:
can test this out when 8213 is PR'd but I think here setting `this.bannerList = banners` should be sufficient if we are receiving an array from the vue service

Reply:
excellent thanks @sam-warren !

Reply:
+1 this.bannerList = banners